tcs-service/src/test/java/com/transformuk/hee/tis/tcs/service/api/PostResource2Test.java

```diff
@@ -260,6 +289,20 @@ public void updatePostShouldReturnUnauth() throws Exception {
         .andExpect(jsonPath("$.message").value("error.accessDenied"));
   }
 
+  @Test
+  public void updatePostShouldPublishPostSavedEvent() throws Exception {
```

Comment:
test name suggests it should verify that an appropriate `PostSavedEvent` was published but instead the HTTP Response is verified. `createPostShouldPublishPostSavedEvent()` verifies both and I'm not sure that's necessary / a useful addition.

Reply:
Ah this is just a test I didn't finish